crud/src/main/java/com/redhat/lightblue/rest/crud/AbstractCrudResource.java

```diff
@@ -93,6 +96,14 @@
         java.security.Security.setProperty("networkaddress.cache.ttl", "30");
     }
 
+    /**
+     * Thread-safe, shared static instance for all requests. Registry is also static, so does not
+     * make much difference to inject. If you wanted to inject, you could do so rather easily using
+     * CDI injection.
+     */
+    private static final RequestMetrics metrics =
+            new DropwizardRequestMetrics(MetricRegistryFactory.getMetricRegistry());
```

Comment:
This should probably move to the RestConfiguration class, where it can pass it to the LightblueFactory.

Also, I forgot that `getMetricRegistry()` also initializes the JMX reporter. We should probably make that clearer somehow. I'd go with simply `getJmxMetricRegistry()`.